projects/BFDR/NatalityRecord_submissionProperties.cs

```diff
@@ -1439,6 +1439,51 @@ protected void SetPlurality(int? value)
                 Subject.MultipleBirth.Extension.Add(plurality);
             }
         }
+    
+        /// <summary>Patient Fetal Death</summary>
+        /// <value></value>
+        /// <example>
+        /// <para>// Setter:</para>
+        /// <para>ExampleBirthRecord.PatientFetalDeath = true;</para>
+        /// <para>// Getter:</para>
+        /// <para>Console.WriteLine($"Patient Fetal Death: {ExampleBirthRecord.PatientFetalDeath}");</para>
+        /// </example>
+        [Property("PatientFetalDeath", Property.Types.Bool, "PatientFetalDeath", "PatientFetalDeath", false, VR.IGURL.PatientFetalDeath, true, 288)]
+        [FHIRPath("Bundle.entry.resource.where($this is Patient)", "deceased")]
+
+        public bool? PatientFetalDeath
```

Comment:
if this field should always be set for fetal death records, maybe it should be called as part of the default fetal death record constructor.